### ISSUE TITLE
MNIST not reproducible 

### ISSUE THREAD

**Issue by ko3n1g** [deepmind/learning-to-learn#33]
Can anyone provide a requirements.txt to make this code work? 
I have been using 
tensorflow-gpu==1.5
dm-sonnet==1.10
numpy==1.14

and Python3.6, but evaluating a MNIST L2L optimizer does not result in the same loss as during training, not even close. 

I have applied the code of #22

**Comment by ko3n1g** [comment]
solved with #34
